decode/performance.ts

```diff
@@ -0,0 +1,105 @@
+import { Event, Token } from "../types/data";
+import { PerformanceEvent } from "../types/decode/performance";
+import { ConnectionData, LargestContentfulData, LongTaskData, MemoryData } from "../types/performance";
+import { NavigationData, NetworkData, PaintData } from "../types/performance";
+
+export function decode(tokens: Token[]): PerformanceEvent  {
+    let time = tokens[0] as number;
+    let event = tokens[1] as Event;
+    switch (event) {
+        case Event.Connection:
+            let connectionData: ConnectionData = {
+                downlink: tokens[2] as number,
+                rtt: tokens[3] as number,
+                saveData: tokens[4] as number,
+                type: tokens[5] as string
+            };
+            return { time, event, data: connectionData };
+        case Event.Contentful:
```

Comment:
>Contentful [](start = 19, length = 10)

Is this for a ContentfulPaint? I'd keep the full name as the shortened form doesn't make sense.

Reply:
Yes, will change. Thanks